internal/eval/eval.go

```diff
@@ -3030,6 +3030,33 @@ func evalHSET(args []string, store *dstore.Store) []byte {
 	return clientio.Encode(numKeys, false)
 }
 
```

Comment:
Could you please add 2-3 lines of comment describing the function on a higher level.
Thanks